src/Courses/TopicSettings/TopicSettingsPage.tsx

```diff
@@ -202,6 +295,61 @@ export const TopicSettingsPage: React.FC<TopicSettingsPageProps> = ({topic: topi
 
     return (
         <Grid container spacing={5} style={{maxWidth: '100%', marginLeft: '0px'}} {...getRootProps({refKey: 'innerRef'})}>
+            <Snackbar
+                anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
+                open={updateAlert.message !== ''}
+                autoHideDuration={6000}
+                onClose={() => setUpdateAlert({message: '', severity: 'info'})}
+                style={{ maxWidth: '50vw' }}
+            >
+                <MUIAlert severity={updateAlert.severity}>
+                    {updateAlert.message}
+                </MUIAlert>
+            </Snackbar>
+            <ConfirmationModal
+                headerContent={<h4>Overwrite settings</h4>}
+                cancelText="No"
+                confirmText="Yes"
+                bodyContent={<>
+                    <h6>Do you want to overwrite the following settings:</h6>
+                    <ul>
+                        {Object.entries(translateTopicSettings(topicSettingsOverwriteModalOptions?.examSettings)).map((setting) => (
+                            // <li key={Math.random()}>TOMTOM</li>
+                            <li key={`${setting[0]}-${setting[1]}`}>
```

Comment:
Do these need to be null-checked, or are they guaranteed cause they come from `entries`?

Reply:
Added a comment